docs/cda_ocat.rst

```diff
@@ -0,0 +1,296 @@
+Chandra Data Archive and Ocat
+=============================
+
+The :mod:`mica.archive.cda.services` module provides a Python interface to the
+Chandra Data Archive (CDA) web services and an interface to a local disk copy of
+the Observation Catalog (Ocat) if that is available.
+
+The CDA web services interface provides access to the following. Note however
+that the CDA is not accessible from the GRETA network.
+
+- Ocat details: full details from the Ocat (124 fields) for the mission, plus
+  access to ACIS windows, roll requirements and time requirements for
+  applicable observations.
+- Ocat summary: summary data from the Ocat (26 fields).
+- Proposal abstracts: abstract information for each observation.
+- Archive file list: list of raw data files in the Chandra archive for each
+  observation.
+
+On the HEAD and GRETA networks, a copy of the Ocat details table (124 fields) is
+maintained and updated daily. This is a compressed HDF5 file which is
+approximately 5 Mb and can be synced to a local (laptop) computer.
+It is located at::
+
+  ${SKA}/data/mica/archive/ocat_target_table.h5
+
+This local file version can be used for faster, network-free queries of the
+Ocat.
+
+This is the type of content that can be seen directly at:
+
+https://cda.harvard.edu/srservices/ocatDetails.do?obsid=2121
+
+Local Ocat access
+-----------------
+
+The :func:`~mica.archive.cda.services.get_ocat_local` function provides fast
+access to the Ocat if the local Ocat HDF5 file is available on disk. In
+particular reading the entire Ocat or querying on a target name substring is at
+least 10 times faster than using the `Web access to CDA`_.
+
+The local version of the Ocat is cached in memory the first time it is read in
+full, so this means subsequent full reads or searches for target names will be
+much faster. The cache expires in one hour in case you have a long-running job.
+
+Any of the fields in the returned table can be used for filtering the query
+via exact matches of the parameters::
+
+   >>> from mica.archive.cda import get_ocat_local
```

Comment:
I think there was some function renaming after the very comprehensive docs?  Because this doesn't work for me.

Reply:
Oops, thought I was working in a brand new checkout of the branch but it was stale.